src/item.cpp

```diff
@@ -2,6 +2,7 @@
 #include "player.h"
 #include "output.h"
 #include "skill.h"
+#include "monster.h"
```

Comment:
Why this include? I don't see any usage of the `monster` class or anything else from that file here.